cli/common/result.go

```diff
@@ -0,0 +1,62 @@
+package common
```

Comment:
It would be great to rename `Res` to `Result`.

Reply:
Everything for you